src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -4,6 +4,32 @@
 
 * Drop support for .NET 6 as this target is no longer supported.
   ([#2159](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2159))
+* The new database semantic conventions can be opted in to by setting
+  the `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable. This allows for a
+  transition period for users to experiment with the new semantic conventions
+  and adapt as necessary. The environment variable supports the following
+  values:
+  * `database` - emit the new, frozen (proposed for stable) database
+  attributes, and stop emitting the old experimental database
+  attributes that the instrumentation emitted previously.
+  * `database/dup` - emit both the old and the frozen (proposed for stable) database
+  attributes, allowing for a more seamless transition.
+  * The default behavior (in the absence of one of these values) is to continue
+  emitting the same database semantic conventions that were emitted in
+  the previous version.
+  * Note: this option will be be removed after the new database
+  semantic conventions is marked stable. At which time this
+  instrumentation can receive a stable release, and the old database
+  semantic conventions will no longer be supported. Refer to the
+  specification for more information regarding the new database
+  semantic conventions for
+  [spans](https://github.com/open-telemetry/semantic-conventions/blob/v1.28.0/docs/database/database-spans.md).
+  ([#2229](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2229))
+* **Breaking change**: The `peer.service` and `server.socket.address` attributes
+  are no longer emitted. Users should rely on the `server.address` attribute
+  for the same information. Note that `server.address` is only included when
+  the `EnableConnectionLevelAttributes` option is enabled.
```

Comment:
NIT: Can it be done as separate, tiny PR? With information that it is updating network sem conv to exact version?
It will be easier to follow changes for end-users.

Reply:
> With information that it is updating network sem conv to exact version?

What do you mean? Are you saying the changelog should reference the version of semantic conventions where these changes were introduced?